python/graphstorm/gsf.py

```diff
@@ -1111,9 +1111,8 @@ def create_task_decoder(task_info, g, decoder_input_dim, train_task):
     elif task_info.task_type in [BUILTIN_TASK_LINK_PREDICTION]:
         return create_builtin_lp_decoder(g, decoder_input_dim, task_info.task_config, train_task)
     elif task_info.task_type in [BUILTIN_TASK_RECONSTRUCT_NODE_FEAT]:
-        return create_builtin_reconstruct_nfeat_decoder(g, decoder_input_dim, task_info.task_config, train_task)
-    elif task_info.task_type in [BUILTIN_TASK_RECONSTRUCT_EDGE_FEAT]:
-        return create_builtin_reconstruct_efeat_decoder(g, decoder_input_dim, task_info.task_config, train_task)
+        return create_builtin_reconstruct_nfeat_decoder(
+            g, decoder_input_dim, task_info.task_config, train_task)
```

Comment:
Why remove the edge feature reconstruction?

Reply:
Ah good catch I'll add this back.